array-foreach/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": false,
```

Comment:
This should be "true". It is not consistent with the one in wiki which is for the "master" branch. The README should be up to date though.